package.json

```diff
@@ -85,13 +85,13 @@
     "test:ci": "npm test -- --coverage --watchAll=false",
     "test:cvg": "npm run test:ci -- --watchAll",
     "lint": "npm run lint:src && npm run lint:cypress",
-		"lint:src": "eslint src --ext .js,.jsx",
-		"lint:cypress": "eslint cypress/integration --ext .js,.jsx",
+    "lint:src": "eslint src --ext .js,.jsx",
+    "lint:cypress": "eslint cypress/integration --ext .js,.jsx",
     "lint:fix": "npm run lint:src:fix && npm run lint:cypress:fix",
-		"lint:src:fix": "eslint src --ext .js,.jsx --fix",
-		"lint:cypress:fix": "eslint cypress/integration --ext .js,.jsx --fix",
-		"lint:diff:develop": "eslint $(git diff --diff-filter=A --name-only develop HEAD '***.js' '***.jsx' | xargs)",
-		"lint:diff:develop:fix": "eslint --fix $(git diff --diff-filter=A --name-only develop HEAD '***.js' '***.jsx' | xargs)",
+    "lint:src:fix": "eslint src --ext .js,.jsx --fix",
+    "lint:cypress:fix": "eslint cypress/integration --ext .js,.jsx --fix",
+    "lint:diff:develop": "eslint $(git diff --diff-filter=A --name-only develop HEAD '***.js' '***.jsx' | xargs)",
+    "lint:diff:develop:fix": "eslint --fix $(git diff --diff-filter=A --name-only develop HEAD '***.js' '***.jsx' | xargs)",
     "plop": "plop --plopfile .plop/index.js",
```

Comment:
Was this file modified intentionally?

Reply:
let me be more direct. Revert this file please.

Reply:
Reverted.